src/mobot/settings.py

```diff
@@ -129,13 +131,16 @@
             'NAME': DATABASE_NAME,
             'USER': DATABASE_USER,
             'PASSWORD': DATABASE_PASSWORD,
-            'HOST': DATABASE_HOST,
```

Comment:
removing this breaks working with postgres in a container.

Reply:
👍